packages/mobile/src/config.ts

```diff
@@ -64,6 +64,10 @@ export const VERIFICATION_STATE_EXPIRY_SECONDS = 30
 
 export const ATTESTATION_REVEAL_TIMEOUT_SECONDS = 60 // 1 minute
 
+// We can safely assume that any balance query returning a number
+// higher than this is incorrect (currently set to 10M)
+export const WALLET_BALANCE_UPPER_BOUND = new BigNumber('1e10')
```

Comment:
If at some point a user files a bug report that "Balance is not displayed correctly on my account with greater than $10M" I indeed agree that will be an interesting day. 😆